app/models/lesson.rb

```diff
@@ -76,14 +83,4 @@ def remove_slug
   def should_generate_new_friendly_id?
     name_changed? || (slug.blank? && !deleted?)
   end
-
-  def update_from_github
-    response = Github.get_content(github_path)
-    if response[:error]
```

Comment:
Why do we not need this error handling any more?

Reply:
The file reading itself will throw an error which should propagate back to the github callbacks controller.